src/cron/views.py

```diff
@@ -32,61 +32,62 @@ def reminders_index(request):
     """
     reminders = models.Reminder.objects.filter(journal=request.journal)
 
-    form = forms.ReminderForm()
-
     if 'delete' in request.POST:
         reminder_id = request.POST.get('delete')
         reminder = get_object_or_404(models.Reminder, pk=reminder_id, journal=request.journal)
         reminder.delete()
         return redirect(reverse('cron_reminders'))
 
-    if request.POST:
-        form = forms.ReminderForm(request.POST)
-
-        if form.is_valid():
-            reminder = form.save(commit=False)
-            reminder.journal = request.journal
-            reminder.template_name = slugify(reminder.template_name)
-            reminder.save()
-
-            check_template = logic.check_template_exists(request, reminder)
-
-            if check_template:
-                return redirect(reverse('cron_reminders'))
-            else:
-                return redirect(reverse('cron_create_template', kwargs={'reminder_id': reminder.pk,
-                                                                        'template_name': reminder.template_name}))
-
     template = 'cron/reminders.html'
     context = {
         'reminders': reminders,
-        'form': form,
     }
 
     return render(request, template, context)
 
 
 @editor_user_required
-def edit_reminder(request, reminder_id):
+def edit_reminder(request, reminder_id=None):
```

Comment:
This change means one can edit a reminder without passing through a reminder_id but that doesn't make a lot of sense. Is this change intended? Are you allowing creation of reminders here? Worth updating the name and the docstring in that case.